dotnet-authserver/src/TeacherIdentity.AuthServer/Infrastructure/Http/HttpRateLimitingException.cs

```diff
@@ -0,0 +1,25 @@
+namespace TeacherIdentity.AuthServer.Infrastructure.Http;
+
+public class HttpRateLimitingException : Exception
```

Comment:
I don't think we need the `Http`-prefix if it's inside an `Http` namespace. `TooManyRequests` matches the spec for 429 too.
```suggestion
public class TooManyRequestsException : Exception
```

Reply:
Done